managed-services/transactional-email/reference-content/understanding-tem-reputation-score.mdx

```diff
@@ -0,0 +1,75 @@
+---
+meta:
+  title: Understanding your reputation score with Transactional Email
+  description: This section provides information about your domains' reputation score
+content:
+  h1:  Understanding your reputation score with Transactional Email
+  paragraph: This section provides information about your domains' reputation score
+tags: transactional-email domain-reputation email-deliverability score reputation-score
+dates:
+  validation: 2023-07-28
+  posted: 2023-07-28
+categories:
+  - managed-services
+---
+
+This documentation provides advice on how to maintain a good [reputation](/managed-services/transactional-email/concepts/#reputation-score) score and helps you understand what you need to do, to have your emails delivered.
+
+<Macro id="iam-requirements" />
+
+
+## Checking your domain reputation
+
+Here is a list of some of the free tools you can use to check and monitor the reputation of your domains:
+
+- [Google Postmaster](https://postmaster.google.com/managedomains) tracks the reputation of your domain with Gmail users and allows you to get insights on how to improve your domain's reputation.
+- [MxToolBox](https://mxtoolbox.com/MXLookup.aspx) allows you to check for any blacklisting and deliverability issues that might be impacting your domains' reputation.
```

Comment:
Sometimes domain is plural (domains') and sometimes singular (domain's) in this doc - I think it should probably always be singular?